Nodejs/Setup/setup.proj

```diff
@@ -19,6 +19,7 @@
     <ProjectFile Include="NodejsTools\NodejsTools.wixproj"/>
     <ProjectFile Include="NodejsToolsInstaller\NodejsToolsInstaller.wixproj"/>
     <ProjectFile Include="..\..\Internal\Nodejs\VsLoggerSetup\VsLogger.wixproj" Condition="'$(IncludeVsLogger)' == 'True'"/>
+    <ProjectFile Include="InstallerTelemetryAction\InstallerTelemetryAction.csproj" Condition="'$(ReleaseBuild)' == 'True'" />
```

Comment:
Is this flag actually set to 'True' here, or should we just check that it's not empty?